App/Views/User/login.html

```diff
@@ -1,65 +1,78 @@
-{% extends "base.html" %}
-
-{% block title %}Connexion{% endblock %}
-
-{% block body %}
-<div class="container-fluid bg-image">
-    <div class="row">
+{% extends "base.html" %} {% block title %}Connexion{% endblock %} {% block body
+    %}
+    <div class="container-fluid bg-image">
+      <div class="row">
         <div class="login-wraper">
-            <div class="hidden-xs">
-                <img src="images/login.jpg" alt="">
+          <div class="hidden-xs">
+            <img src="images/login.jpg" alt="" />
+          </div>
+          <div class="banner-text hidden-xs">
+            <div class="line"></div>
+            <div class="b-text">
+              Des <span class="color-active">centaines<br /> </span>
+              <span class="color-b1">d'articles vous </span>attendent
             </div>
-            <div class="banner-text hidden-xs">
-                <div class="line"></div>
-                <div class="b-text">
-                    Des <span class="color-active">centaines<br> </span> <span class="color-b1">d'articles vous
-                </span>attendent
+          </div>
+          <div class="login-window">
+            <div class="l-head">Connexion</div>
+            <div class="l-form">
+              <form action="" method="post">
+                <div class="form-group">
+                  <label for="exampleInputEmail1">Email</label>
+                  <input
+                    type="email"
+                    class="form-control"
+                    id="exampleInputEmail1"
+                    name="email"
+                    placeholder="sample@gmail.com"
+                  />
                 </div>
-            </div>
-            <div class="login-window">
-                <div class="l-head">
-                    Connexion
+                <div class="form-group">
+                  <label for="exampleInputPassword1">Mot de passe</label>
+                  <input
+                    type="password"
+                    class="form-control"
+                    id="exampleInputPassword1"
+                    name="password"
+                    placeholder="**********"
```

Comment:
J'aurais plutot mis ca qu'est ce que t'en penses ? :


```suggestion
                    placeholder="Saisir votre mot de passe"
```